cypress/integration/jobs/update-job.spec.js

```diff
@@ -0,0 +1,165 @@
+/// <reference types="cypress" />
+import 'cypress-audit/commands'
+
+const TimeToWaitForRenderToFinish = 5000
```

Comment:
Does adding a default max wait time to cypress.json not work?

Reply:
unfortunately not , because the problem is not time out, the problem is detached from DOM